### PR TITLE
gguf: sync type with upstream llama.cpp

### PR DESCRIPTION
**Pull request by ngxson**
Generated by `scripts/generate-llm.ts`

### CODE REVIEW

**Review by Vaibhavs10 — COMMENTED** [review]
GG! 🔥

File: packages/gguf/src/transformer-llm.ts
Comment (julien-c) [inline packages/gguf/src/transformer-llm.ts:65]: they dropped this arch?

**Review by julien-c — APPROVED** [review]
lgtm, thanks for updating!

File: packages/gguf/src/transformer-llm.ts
Comment (ngxson) [reply packages/gguf/src/transformer-llm.ts:65]: Yes it's removed in https://github.com/ggerganov/llama.cpp/pull/7408

Btw, should I merge it now, or wait for approval from @mishig25 ? Thanks.

Comment (julien-c) [reply packages/gguf/src/transformer-llm.ts:65]: @mishig25 is OOO so you can merge 🔥

**Comment by julien-c** [comment]
note that you will need to trigger a release of `@huggingface/gguf` – do you know how to do it?

**Comment by ngxson** [comment]
No I don't know about that. Could you give me a guide? Thanks.

**Comment by julien-c** [comment]
go to the [Actions tab](https://github.com/huggingface/huggingface.js/actions), then click on the relevant package's "Version and release" action, then "Run workflow" – you'll need to specify patch|minor|major

cc @coyotte508 who built those actions

Let me know if this works!

**Comment by ngxson** [comment]
Yes that works! 🎉 🎉  
I published `v1.8.0` --> https://www.npmjs.com/package/@huggingface/gguf?activeTab=versions

Thank you @julien-c @coyotte508 !

**Pull request merged**
